GDPR/Utilities/String+Duplication.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Stefan Draskovits on 29.11.22.
+//
+
+import Foundation
+
+extension Sequence where Element: Hashable {
+    var hasDuplicates: Bool {
+        var elements = Set<Element>()
+        for element in self {
```

Comment:
couldn't we just put self into a Set and then compare the count of self against the count of the Set, and if the Set count is smaller, we know that there have to be duplicates?

Reply:
Yes, your solution would work as well, but does it matter here? 

My solution has a better best case performance (e.g. [1,1,2,3....]) 
In worst case my solution is 2x calculating hash values, whereas your solution is 1x calculating hash values + 2x count.